docs/k8s-cluster/nvidia-network-operator.md

```diff
@@ -0,0 +1,233 @@
+Deploy Nvidia Network Operator with DeepOps
+===========================================
+
+## Overview
+
+   Nvidia Network Operator leverages Kubernetes CRDs and Operator SDK to manage networking related components in Kuberenets cluster. One of the key components is SR-IOV, which partitions a single PCIe hardware into multiple Virtual Functions (VFs) and attach them directly to Kubernetes pods without going through the virtualization layer on the hosts, thus enables the high performance communication between workloads. High performance networking in Kuberentes also requires a few other components, such as multus-CNI, device drivers and plugins, etc, Nvidia network operator aims to manage all those necessary components automatically under one operator frame work to simply the deployment, operation and management of Nvidia networking for Kubernetes.
+
+Here are the key components that Nvidia network operator try to deploy together:
+
+* SR-IOV Virtual Function (VF) activation
+* Multus CNI
+* SR-IOV CNI for kubernetes
+* SR-IOV device plugin for kubernetes
+* Multus CNI
+* Helm chart for Nvidia network operator
+
+This playbook also install the latest Kubeflow/MPI-Operator for multi-node MPI jobs.

```

Comment:
Does it install the "latest" or a specific version? We should peg version and document that version here.